src/components/Viewer.jsx

```diff
@@ -265,7 +265,7 @@ export class Viewer extends Component {
     }
     return (
       <div className={styles['pho-viewer-wrapper']} role='viewer' ref={viewer => { this.viewer = viewer }}>
-        <ViewerToolbar hidden={hideToolbar} />
+        <ViewerToolbar hidden={hideToolbar} currentPhoto={currentPhoto} />
```

Comment:
I summon @goldoraf for that: do we want ViewerToolbar to be a connected component? And grabbing the currentPhoto from the global state instead of its parent?